summary.qmd

```diff
@@ -4,35 +4,39 @@ title: "What is Open Source?"
 
 ## What is Open Source?
 
-When someone says 'open source' - what does that mean to you?\
-What are the important characteristics of something that is regarded as 'open source'?
+The [Open Source Initiative](https://opensource.org/osd){target="_blank"} delineates a number of characteristics of what it means to be **open source**, including free distribution, available source code, and the ability to create derived works.
 
--   Does the cost matter?
--   Does the ability to review the code matter?
--   Does the ability to reuse the code matter?
--   Does the ability to modify the code matter?
+According to [Merriam-Webster](https://www.merriam-webster.com/dictionary/open-source){target="_blank"}, open source is “having the source code freely available for possible modification and redistribution”.
 
-## How to Contribute
+[Wikipedia](https://en.wikipedia.org/wiki/Open-source_software){target="_blank"} defines open-source software (OSS) as a computer software that is released under a license in which the copyright holder grants users the rights to use, study, change and distribute the software and its source code to anyone and for any purpose.
 
-Contribute to the discussion here in GitHub Discussions:\
-[What is open source?](https://github.com/phuse-org/OSTCDA/discussions/1){target="_blank"}
+All three of these sources refer to the ability to see the code, use the code, change the code, and share the code. And to be able to do so without any cost.
 
-## Guidance
+### How is this enabled?
 
--   Provide your thoughts and perspectives
+Fundamentally, the code is owned by someone - by some entity, whether an individual, an organization, or a group of other entities. The owner of the code have the right to disseminate the code openly and describe how it can be used by others. This is accomplished through a license agreement (discussed in another section). For any piece of **open source code**, the owner of that code has decided to release it publicly with an associated license that defines how others may use or modify the code.
```

Comment:
license and potentially a code of conduct. Code of conduct statements are more common in recent open-source software repositories. This is to encourage any potential contributors and developers to work in a constructive, ethical and respectful way.